archeryutils/targets.py

```diff
@@ -59,42 +63,24 @@ def __init__(
                 f"""Invalid Target Face Type specified.\n"""
                 f"""Please select from '{"', '".join(systems)}'."""
             )
-
-        if native_dist_unit in (
-            "Yard",
-            "yard",
-            "Yards",
-            "yards",
-            "Y",
-            "y",
-            "Yd",
-            "yd",
-            "Yds",
-            "yds",
-        ):
-            native_dist_unit = "yard"
-        elif native_dist_unit in (
-            "Metre",
-            "metre",
-            "Metres",
-            "metres",
-            "M",
-            "m",
-            "Ms",
-            "ms",
-        ):
-            native_dist_unit = "metre"
-        else:
+        if native_dist_unit not in Length.yard | Length.metre:
             raise ValueError(
                 f"Distance unit '{native_dist_unit}' not recognised. "
                 "Select from 'yard' or 'metre'."
             )
+        distance = Length.to_metres(distance, native_dist_unit)
 
+        if native_diameter_unit not in Length.cm | Length.inch | Length.metre:
```

Comment:
Please use logical `or` rather than bitwise `|`.

Reply:
Means different things? Logical or would be ~slightly~ a lot more verbose:
```python
# have to check membership of each unit set seperately using logical ors
if native_diameter_unit not in Length.cm or native_diameter_unit not in Length.inch or native_diameter_unit not in Length.metre:
    ...

# bitwise or creates union of sets of aliases, equivalent to:
if native_diameter_unit not in set.union(Length.cm, Length.inch, Length.metre):
    ...

# parens for emphasis, this is definitely not what we want
if (native_diameter_unit not in Length.cm) or Length.inch or Length.metre
    # returns true if variable is in cm or otherwise returns the inch set
```

Reply:
Huh. TIL: Certain operators are overloaded in python to perfom set logic.
So happy to leave as is. :)

In general bitwise logic operators are intended for integers only, and [lead to some gotchas](https://stackoverflow.com/questions/3845018/boolean-operators-vs-bitwise-operators) if applied inappropriately.